src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,55 @@
+package duke;
```

Comment:
Perhaps split classes into multiple packages other than just the single duke package.